otherlibs/runtime_events/runtime_events.mli

```diff
@@ -59,6 +59,13 @@ type runtime_counter =
 | EV_C_REQUEST_MINOR_REALLOC_REF_TABLE
 | EV_C_REQUEST_MINOR_REALLOC_EPHE_REF_TABLE
 | EV_C_REQUEST_MINOR_REALLOC_CUSTOM_TABLE
+| EV_C_MAJOR_HEAP_POOL_WORDS
+| EV_C_MAJOR_HEAP_POOL_LIVE_WORDS
+| EV_C_MAJOR_HEAP_LARGE_WORDS
+| EV_C_MAJOR_HEAP_POOL_FRAG_WORDS
+| EV_C_MAJOR_HEAP_POOL_LIVE_BLOCKS
+| EV_C_MAJOR_HEAP_LARGE_BLOCKS
+| EV_C_MINOR_HEAP_SIZE_WORDS
```

Comment:
Hmm - is this a point to consider whether `runtime_counter` ought to be an extensible variant?

At a minimum, these need `@since` markings.

Reply:
Yes, good point on the since annotations I will add those.

I'm not opposed to turning these in to an extensible variant but I don't know what that will do for the C API. Are there any examples in the compiler for similar things?